docs/source/installation.rst

```diff
@@ -169,18 +199,96 @@ However for basic production deployment, you might need to run Uvicorn independe
 
 For more options about Uvicorn (concurrency, SSL etc..) refer: https://www.uvicorn.org/#command-line-options
 
+
+.. _Visualization Tools:
+
+Guide of Visualization Tools
+================================
+
+MONAI Label now supports several mostly adopted open-source viewer software for radiology and pathology.
```

Comment:
consider removing 'mostly'. 'Severak' and 'mostly' are the same

Reply:
Correct, in actual English it shall be like;
`MONAI Label supports the most adopted open-source viewers for Radiology and Pathology.`

Reply:
Thanks, fixed.